python/lsst/sims/photUtils/PhotometricParameters.py

```diff
@@ -2,50 +2,177 @@
 
 __all__ = ["PhotometricParameters"]
 
+class DefaultPhotometricParameters:
+    """
+    This class will just contain a bunch of dict which store
+    the default PhotometricParameters for LSST Bandpasses
+
+    Users should not access this class (which is why it is
+    not included in the __all__ declaration for this file).
+
+    It is only used to initialize PhotometricParameters off of
+    a bandpass name.
+    """
```

Comment:
Maybe make a comment about things which will be the same across bands but are stored in dictionaries anyway (making access easier, although it may look funny).